tests/neo-vm.Tests/Tests/Others/Init.json

```diff
@@ -0,0 +1,47 @@
+﻿{
+    "category": "Others",
+    "name": "Init",
+    "tests":
+    [
+        {
+            "name": "Init script",
+            "script": "0x66",
+            "trigger": "Application",
+            "steps":
+            [
+                {
+                    "state":
+                    {
+                        "state": "Break",
+                        "invocationStack":
+                        [
+                            {
+                                "scriptHash": "0x9B4A30911C6BCD2921CD153A2956DA397B5793A4",
+                                "instructionPointer": 0,
+                                "nextInstruction": "RET"
+                            }
+                        ]
+                    }
+                }
+            ]
+        },
+        {
+            "name": "Init script",
+            "script": "0x66",
+            "trigger": "Application",
+            "steps":
+            [
+                {
+                    "actions":
+                    [
+                        "StepInto"
+                    ],
+                    "state":
+                    {
+                        "state": "Break|Halt"
```

Comment:
`state` nested `state`?

Reply:
Fully agree Erik. Perhaps change the outer "state" to "finalState"? The most inner one looks nice as "state".

Reply:
result?